src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -176,7 +176,7 @@ public sealed class Arguments : ISupportSdcaClassificationLossFactory, ISupportC
         private const Float Threshold = 0.5f;
         private readonly Float _margin;
 
-        internal HingeLoss(Arguments args)
+        internal HingeLoss(Options args)
```

Comment:
>args [](start = 35, length = 4)

Same here, I guess this is a comment that applies to all the files where you rename Arguments to Options.

Reply:
fixed.

fyi.. in this PR the focus is scrubbing the public surface area for  `Options options` .   

There are still  lots of  `internal`  occurrences of `Arguments args` .. Those  are  P2  since they can be renamed post V1.0 as well

---
In reply to: [257145148](https://github.com/dotnet/machinelearning/pull/2563#discussion_r257145148) [](ancestors = 257145148)